lib/service_actor/checks/inclusion_check.rb

```diff
@@ -67,6 +67,7 @@ def check
 
   def define_inclusion_and_message
     if @inclusion.is_a?(Hash)
+      @inclusion[:message] = @inclusion[:message] || DEFAULT_MESSAGE
```

Comment:
This can also be simplified to:

```suggestion
      @inclusion[:message] ||= DEFAULT_MESSAGE
```